src/app/chip_data_model.gni

```diff
@@ -256,6 +254,14 @@ template("chip_data_model") {
           "${_app_root}/clusters/scenes-server/ExtensionFieldSetsImpl.cpp",
           "${_app_root}/clusters/scenes-server/SceneTableImpl.cpp",
         ]
+      } else if (cluster == "icd-management-server") {
+        sources += [
+          "${_app_root}/clusters/icd-management-server/icd-management-server.cpp",
+          "${_app_root}/icd/IcdManagementServer.cpp",
+          "${_app_root}/icd/IcdManagementServer.h",
+          "${_app_root}/icd/IcdMonitoringTable.cpp",
```

Comment:
Shouldn't the IcdMonitoringTable.cpp/h be part of the `source_set("manager-srcs")` (src/app/icd/BUILD.gn).

the ICDManager will need the IcdMonitoring Table and I think the `icd-management-server `can have a dep from `manager-srcs` but not the other way around

Reply:
Moved to manager-srcs.